SECURITY.md

```diff
@@ -0,0 +1,59 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+<!-- BEGIN PRAVEGA SECURITY.MD V0.0.1 BLOCK -->
+
+## Security
+
+Pravega takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations.
+
+If you believe you have found a security vulnerability in any Pravega owned repository, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Pravega Security Team via email(mailto:security@pravega.io).
+
+You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message.
+
+Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
+
+  * Product name and version containing the vulnerability
+  * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+  * Full paths of source file(s) related to the manifestation of the issue
+  * The location of the affected source code (tag/branch/commit or direct URL)
+  * Any special configuration required to reproduce the issue
+  * Step-by-step instructions to reproduce the issue
+  * Proof-of-concept or exploit code (if possible)
+  * Impact of the issue, including how an attacker might exploit the issue
+
+This information will help us triage your report more quickly.
+
+
+## Preferred Languages
+
+We prefer all communications to be in English.
+
+## Policy
+
+Pravega follows the principle of [Coordinated Vulnerability Disclosure](https://www.cisa.gov/coordinated-vulnerability-disclosure-process).
+
+<!-- END PRAVEGA SECURITY.MD BLOCK -->
+
+## Released Versions
+The following versions of Pravega open source are released. Please visit the [Releases](https://github.com/pravega/pravega/releases) page for tags and changelists. 
+
+| Version | Released          |
```

Comment:
It'd be hard to maintain the versions here. Just referencing the releases page should be enough.